pyproject.toml

```diff
@@ -5,6 +5,46 @@ build-backend = "setuptools.build_meta"
 [project]
 dynamic = ["version"]
 name = "openlibrary-client"
+authors = [{name = "Internet Archive", email = "mek@archive.org"}]
+license = {text = "LICENSE"}
+description = "A Python client for Open Library"
+readme = "README.md"
+classifiers = [
+    "Development Status :: 2 - Pre-Alpha",
```

Comment:
'Pre-alpha'? That may be true, but perhaps it should be reviewed.

Reply:
@mekarpeles Please review this comment and the one below.